tools/src/main/scala/com/nvidia/spark/rapids/tool/planparser/SQLPlanParser.scala

```diff
@@ -268,6 +268,25 @@ object SQLPlanParser extends Logging {
     funcName
   }
 
+  def parseProjectExpressions(exprStr: String): Array[String] = {
+    val parsedExpressions = ArrayBuffer[String]()
+    // remove the alias names before parsing
+    val pattern = """(AS) ([(\w# )]+)""".r
+    // This is to split multiple column names in Project. Project may have a function on a column.
+    // This will contain array of columns names specified in ProjectExec.
+    val paranRemoved = pattern.replaceAllIn(exprStr.replace("),", "::"), "")
```

Comment:
lets start putting more description on these so when we come back to them we can understand.... put example as well.